src/tools/gophish_cleaner.py

```diff
@@ -2,23 +2,25 @@
 
 """GoPhish Cleaner for a Phishing Campaign Assessment (PCA).
 
+A tool to remove aspects of an Assessment or a campaign from GoPhish.
+
 Usage:
-  gophish-cleaner (--assessment | --campaigns | --smtp | --pages | --groups | --templates) [--log-level=LEVEL] ASSESSMENT_ID SERVER API_KEY
+  gophish-cleaner (--assessment | --campaigns | --groups | --pages | --smtp | --templates) [--log-level=LEVEL] ASSESSMENT_ID SERVER API_KEY
   gophish-cleaner (-h | --help)
   gophish-cleaner --version
 
 Options:
-  SERVER 	--> Full URL to GoPhish server
-  API_KEY 	--> API Access Key
-  ASSESSMENT_ID      Assessment ID to remove.
-  -a --assessment   Remove the entire assessment.
-  -c --campaigns   Remove only campaigns
-  -s --smtp    Remove only SMTP
-  -p --pages    Remove only pages
-  -g --groups    Remove only groups
-  -t --templates    Remove only templates
-  -h --help      Show this screen.
-  --version      Show version.
+  API_KEY                   API Access Key.
+  ASSESSMENT_ID             Assessment ID to remove.
+  SERVER                    Full URL to GoPhish server.
+  -a --assessment           Remove the entire assessment.
+  -c --campaigns            Remove only campaigns.
```

Comment:
`Remove the entire assessment.` --> `Remove all data for the specified assessment.`
`Remove only campaigns.` --> `Remove all campaigns from the specified assessment.`
etc...